calico/felix/frules.py

```diff
@@ -538,7 +538,7 @@ def update_ipsets(type,
     """
     for rule in rule_list:
         if rule.get('cidr') is None:
-            log.error("Invalid %s rule without cidr for %s : %s",
+            log.error("Invalid %s rule without cidr for %s : %s" %

```

Comment:
Thanks. So we have our policy decided then! I'll be consistent from now on.

I've just added testing for gazillions of logs, so I am no longer so terrified of changing them and breaking things.